### PR TITLE
feat: use generated displayName for alerts without a custom name 

### PR DESCRIPTION
**Pull request by dblandin**
The type of this PR is: **Feat**

Jira: [ONYX-227](https://artsyproduct.atlassian.net/browse/ONYX-227)

### Description

Allow collectors to create an alert without specifying a custom name and fallback to our generated display name which automatically displays the most relevant criteria at a glance.

With the feature flag enabled, the "Alert Name" input placeholder will be empty, and creating/updating an alert with no custom name will trigger the new generated default/fallback name.

**Next Steps**
- Open a similar PR for the app/eigen
- Update the create/edit flows to utilize the previewSavedSearch query field in Metaphysics so we can preview the generated name and formatted pills without formatting client-side.

### Screen Recording

https://www.loom.com/share/6ccad310d13248e5a66e9b0aff289eb9?sid=aa946cc3-ac57-490c-91b0-af74962d6747

[ONYX-227]: https://artsyproduct.atlassian.net/browse/ONYX-227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by nickskalkin — DISMISSED** [review]
Look good to me 👍

File: src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx [hunk 1]
```diff
@@ -160,7 +166,11 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
               <Input
                 title="Alert Name"
                 name="name"
-                placeholder={entity.placeholder}
+                placeholder={
+                  isFallbackToGeneartedAlertNamesEnabled
```
Comment (olerichter00) [inline src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx:170]:
```suggestion
                  isFallbackToGeneratedAlertNamesEnabled
```

**Review by olerichter00 — DISMISSED** [review]
🌟

File: src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx (hunk 1 above)
Comment (dblandin) [reply src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx:170]: Ahh, good catch. Fix one typo and introduce another! 😅

**Pull request merged**